torchmetrics/functional/text/bert.py

```diff
@@ -449,9 +451,16 @@ def _rescale_metrics_with_baseline(
     return all_metrics[..., 0], all_metrics[..., 1], all_metrics[..., 2]
 
 
+@deprecated(
+    args_mapping={"predictions": "preds", "references": "target"},
+    target=True,
+    stream=logging.warning,
```

Comment:
I think we want to keep the default `DeprecationWarning` or #744

Reply:
@stancld ^^

Reply:
Will have a look :]